guides/v2.3/graphql/tutorials/checkout/checkout-shipping-method.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: tutorial
+group: graphql
+title: Step 6. Set shipping method
+subtitle: GraphQL checkout tutorial
+level3_subgroup: graphql-checkout
+return_to:
+  title: GraphQL Overview
+  url: graphql/index.html
+menu_order: 60
+functional_areas:
+  - Integration
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+`setShippingMethodsOnCart` mutation query is using to define a shipping method for your order and requires the next input parameters:
+ - `cart_id`
+ - `carrier_code`
+ - `method_code`
+
```

Comment:
Add this below:

{{ CART_ID }}` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html).

Reply:
**done**